bskyweb/templates/starterpack.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block html_head_extra -%}
+  <meta name="apple-itunes-app" content="app-id=xyz.blueskyweb.app, app-clip-bundle-id=xyz.blueskyweb.app.AppClip, app-clip-display=card">
```

Comment:
🙏

Reply:
just checking, is this only being added on the `/start/*` route?

Reply:
Yeah that's right, this template is only wired-up to the new `server.WebStarterPack` handler for that route.  The routing looks like this:
```go
e.GET("/start/:handleOrDID/:rkey", server.WebStarterPack)
```